internal/githubprovider/model.go

```diff
@@ -139,11 +142,28 @@ func fromIssue(batch *dvmodel.Batch, input *github.Issue) error {
 			return fmt.Errorf("unsupported pmbodyparser.Kind: %v", relationship.Kind)
 		}
 	}
-
 	batch.Tasks = append(batch.Tasks, &issue)
 	return nil
 }
 
+func parseDuration(body string) string {
+	rules := regexp.MustCompile(`\b\w+\b`)
+	words := rules.FindAllString(body, -1)
+	previous := ""
+	for _, v := range words {
+		if previous == "time" {
+			_, err := str2duration.ParseDuration(v)
+			if err != nil {
+				fmt.Println(err)
+				return "undefined"
```

Comment:
```suggestion
				const InvalidDuration string = "invalid" 				
				return InvalidDuration
```

"Invalid" is better than "undefined" in this case.

Creating a `const` for strings is better to allow checking from other packages.

(The `const` should be moved outside of the function.)